assets/scripts/app/keyboard_commands.js

```diff
@@ -110,24 +112,40 @@ function getHoveredEl () {
   return el
 }
 
-// In case anyone tries a save shortcut key out of reflex,
-// we inform the user that it's not necessary.
-registerKeypress('ctrl s', {
-  trackAction: 'Command-S or Ctrl-S save shortcut key pressed'
-}, function () {
-  showStatusMessage(msg('STATUS_NO_NEED_TO_SAVE'))
-})
+export function registerKeypresses () {
+  // In case anyone tries a save shortcut key out of reflex,
+  // we inform the user that it's not necessary.
+  registerKeypress('ctrl s', {
+    trackAction: 'Command-S or Ctrl-S save shortcut key pressed'
+  }, function () {
+    showStatusMessage(msg('STATUS_NO_NEED_TO_SAVE'))
+  })
 
-// Catch-all for the Ctrl-S shortcut from ever trying to
-// save the page contents
-registerKeypress('ctrl s', {
-  preventDefault: true,
-  requireFocusOnBody: false
-}, _.noop)
+  // Catch-all for the Ctrl-S shortcut from ever trying to
+  // save the page contents
+  registerKeypress('ctrl s', {
+    preventDefault: true,
+    requireFocusOnBody: false
+  }, _.noop)
 
-// Catch-all for the backspace or delete buttons to prevent
-// browsers from going back in history
-registerKeypress(['backspace', 'delete'], {
-  preventDefault: true,
-  requireFocusOnBody: true
-}, _.noop)
+  // Catch-all for the backspace or delete buttons to prevent
+  // browsers from going back in history
+  registerKeypress(['backspace', 'delete'], {
+    preventDefault: true,
+    requireFocusOnBody: true
+  }, _.noop)
+
+  // Register keyboard input for show (shift-D)
+  registerKeypress('shift d', showDebugInfo)
```

Comment:
I understand why this got moved here (and we still have a bunch of code in which event listeners are added all at once on behalf of other modules), but I think it might be better for the modules that implement specific functionality to also be in charge of registering their keyboard commands. This is why debug info and infobubble register their own keyboard commands -- it's so that it's much clearer when looking at each module where those listeners are coming from.

This is also helpful to think about in terms of React components. If we were to migrate each of these to components, for example, setting these keypresses will be under their respective `constructor()` or `componentDidMount()` methods. They wouldn't be attached in the module responsible for listening for keystrokes in an abstract fashion.

So why is there a central module for registering keypresses? This probably was the wrong way to do it if it gave the wrong impression. The idea is that some of these are "global" to the application and not attached to specific components. Perhaps if we had a `<App />` component then it would be more obvious if these were registered within that instead.